lib/rules/disallow-multiple-spaces.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Disallows multiple spaces between identifiers, keywords, and any other token
+ *
+ * Type: `Boolean`
+ *
+ * Values: `true`
+ *
+ * #### Example
+ *
+ * ```js
+ * "disallowMultipleSpaces": true
+ * ```
+ *
+ * ##### Valid
+ * ```js
+ * var x = "hello";
+ * function y() {}
+ * ```
+ *
+ * ##### Invalid
+ * ```js
+ * var x  = "hello";
+ * function y() {}
+ * ```
+ */
+
+var assert = require('assert');
+
+module.exports = function() {};
+
+module.exports.prototype = {
+
+    configure: function(disallowMultipleSpaces) {
+        assert(
+            typeof disallowMultipleSpaces === 'boolean',
+            'disallowMultipleSpaces option requires boolean value'
+        );
+        assert(
+            disallowMultipleSpaces === true,
+            'disallowMultipleSpaces option requires true value or should be removed'
+        );
+    },
+
+    getOptionName: function() {
+        return 'disallowMultipleSpaces';
+    },
+
+    check: function(file, errors) {
+        file.getTokens().forEach(function(token, index, tokens) {
```

Comment:
I'm not sure but `getTokens` returns token list without comments and tests suggested by me will fail.
https://github.com/jscs-dev/node-jscs/blob/master/lib/js-file.js#L492

I suggest to use here `while (token = file.nextToken({includeComments: true}) { ...` or something similar to iterate through tokens and comments.